cmd/cli/kubectl-kyverno/apply/apply_command_test.go

```diff
@@ -176,8 +176,8 @@ func Test_Apply(t *testing.T) {
 		},
 		{
 			config: ApplyCommandConfig{
-				PolicyPaths:   []string{"../../../../test/cli/test-validating-admission-policy/validate-deployment/policy.yaml"},
-				ResourcePaths: []string{"../../../../test/cli/test-validating-admission-policy/validate-deployment/deployment1.yaml"},
+				PolicyPaths:   []string{"../../../../test/cli/test-vap/check-deployments-replica/policy.yaml"},
+				ResourcePaths: []string{"../../../../test/cli/test-vap/check-deployments-replica/deployment1.yaml"},
```

Comment:
may be best to use the full name here, for clarity

Reply:
I created a [PR](https://github.com/kyverno/kyverno/pull/7929) to fix it. Thank you!